backport/backport.ts

```diff
@@ -374,6 +374,17 @@ const backport = async ({
 
 	if (!merged) {
 		console.log('PR not merged')
+		for await (const cmt of issue.getComments()) {
+			if (cmt.at.toString().indexOf('This PR must be merged before a backport PR will be created.') >= 0) {
```

Comment:
What does `cmt.at` refer to? Shouldn't it be `cmd.body`? Not sure, but thought cmt type is `Comment` https://github.com/grafana/grafana-github-actions/blob/main/api/api.ts#L95 🤔

Reply:
Ohhhh very good point

Reply:
Fair, I was just digging around the repo for anything using `getComments` but there was very little. However, it looks like the iterator, and thus each instance of `cmt`, is a `[]Comment`. 

Without a nice way to test this I'm just stumbling around in the dark unfortunately. We don't seem to have any existing tests that read the contents of a PR. I've test GH actions themselves before via [act](https://github.com/nektos/act) but I have essentially 0 experience with JS/TS.

Reply:
I'll try to test this in my setup :) Let's see.

Reply:
Looking good! I've done the following:

1. Create a PR without backport labels
2. Add one backport label
3. Comment is made
4. Add another backport label
5. No additional comment is made
6. Delete comment & add another backport label
7. Comment is created again

<img width="937" alt="Screenshot 2024-01-12 at 08 53 55" src="https://github.com/grafana/grafana-github-actions/assets/3782/679436df-236a-4866-96ca-a7231ef98fd0">

Reply:
:+1: Awesome, thanks @zerok. Can I ask how you set thing up to test like that?

Going back to @ifrost comment I think because the return value of `getComments` is an iterator using the for loop and `at` within there is the correct thing to do, but if I have a way to test out changes myself I'm happy to play around with changing things to use `.body` directly if that makes more sense semantically.

Reply:
@cstyan I just have an empty repository with branches like v10.0.x, v10.1.x etc. and I've copied the backports.yml from grafana/grafana into it, just modifying where it fetches the action from. Then I can just mess around with that repository, rebase as needed etc.

Reply:
Perfect, thank you :+1: I'll keep that in mind for the future.

if there are no further concerns, can we merge this?

Reply:
Looks good to me 🙂 Worst case, we can quickly roll it back if necessary.